packages/node_modules/@ciscospark/helper-html/test/unit/spec/html.js

```diff
@@ -109,7 +109,7 @@ skipInNode(describe)(`html`, () => {
     },
     {
       it: `filters javascript: from a href`,
-      input: `<p><a href="javascript:window.close(); return false">click here</a></p>`,
```

Comment:
Could we keep the original test case with an all lowercase test and make these mixed case tests a new test?